### PR TITLE
[FX-5524] Refactor Carousel to Tailwind

### PR DESCRIPTION
**Pull request by augustobmoura**
[FX-5524]

### Description

Replacing MUI styles with Tailwind classes

### How to test

- [Temploy](https://picasso.toptal.net/fx-5524-carousel)
- Check temploy and visual tests

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests ([visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md) included)

**Breaking change**

- [x] codemod is created and showcased in the changeset
- [x] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-5524]: https://toptal-core.atlassian.net/browse/FX-5524?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by augustobmoura** [comment]
@toptal-anvil ping reviewers

**Review by AdrianContiu — APPROVED** [review]
Great job!🚀 Everything works as supposed. I just left a comment about the changeset, please decide if it's relevant.

**Comment by augustobmoura** [comment]
@toptal-anvil ping reviewers

File: .changeset/late-swans-taste.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso-carousel': major
```
[the remark this answers is not in the document]
Comment (augustobmoura) [reply .changeset/late-swans-taste.md:2]: Makes sense, I will add it :+1: 
Thanks for pointing that out

**Comment by augustobmoura** [comment]
@toptal-anvil ping reviewers

File: packages/base/Carousel/package.json [hunk 2]
```diff
@@ -27,16 +27,19 @@
...
   },
+  "devDependencies": {
+    "@toptal/picasso-tailwind-merge": "1.2.0"
+  },
   "sideEffects": [
     "**/styles.ts",
     "**/styles.js"
   ],
   "peerDependencies": {
     "@material-ui/core": "4.12.4",
```
Comment (mkrl) [inline packages/base/Carousel/package.json:40]: Shouldn't `"@material-ui/core"` be removed?

**Review by mkrl — APPROVED** [review]
Looks simple enough. Visual tests and temploy looks good. 
Left one comment that probably has to be addressed, but the rest is good to go

File: packages/base/Carousel/package.json (hunk 2 above)
Comment (augustobmoura) [reply packages/base/Carousel/package.json]: Yep, removed it, good catch!

**Pull request merged**